certified-connectors/TractionGuest/apiProperties.json

```diff
@@ -0,0 +1,34 @@
+{
+  "properties": {
+    "capabilities": [],
+    "connectionParameters": {
+      "token": {
+        "oAuthSettings": {
+          "clientId": "a0d334f16a54c5a979c8851029150e8182322f7de0c09a5e5f2dba83ffbb3321",
```

Comment:
@jzhang729 Could you please replace the ClientId value and dummy value, like below:
"clientId": "[DUMMY_VALUE]"

Reply:
@sriyen-msft Made the fix. Thanks for the feedback.